test('test', () => {});
test('this should fail', () => {
  throw new Error('failure');
});
